bin/plugin/cli.js

```diff
@@ -31,7 +31,7 @@ program
 	.command( 'release-plugin-rc' )
 	.alias( 'rc' )
 	.description(
-		'Release an RC version of the plugin (supports only rc.1 for now)'
+		'Release an RC version of the plugin (will increment the RC number if one already exists for the last version)'
```

Comment:
We can probably drop the note in parens altogether, I think it's kinda implied, and we're doing the same for stable versions where we also don't mention the increment.

```suggestion
		'Release an RC version of the plugin'
```